doc/sphinx/run.rst

```diff
@@ -228,7 +229,7 @@ The parameter :math:`\Gamma` proportional to the reaction constant. It is define
 
 .. math::
 
-   \Gamma = \prod_i \Bigl(\frac{N_i}{V} \Bigr)^{\bar\nu} = V^{-\bar\nu} \prod_i N_i^{\bar\nu_i}
```

Comment:
Maybe we add an explanation for N_i: N_i is here the number of particles of type i *after* the system has equilibrated. Otherwise one might confuse it with the current number of particles which are in the simulation box.